tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+import boto3
+import pytest
+from service import _add_replication
+from moto import mock_s3
+
+
+@mock_s3
+def test_replication(client, logging_bucket):
+    """
+    Test that adding replication to a bucket configures a dr bucket correctly
+    """
+    logging_bucket()
+    bucket_name = 'sd-00000000'
+
+    client = boto3.client('s3')
+    bucket = client.create_bucket(ACL='private', Bucket=bucket_name)
```

Comment:
Do you clean up this bucket somewhere?

Reply:
This is a test that uses a mocked out s3, so no real infra ever gets created.